app/src/main/java/com/example/cm/ui/adapters/SwipeToDelete.java

```diff
@@ -18,6 +27,28 @@ public boolean onMove(@NonNull RecyclerView recyclerView, @NonNull RecyclerView.
         return false;
     }
 
+    @SuppressLint("UseCompatLoadingForDrawables")
+    @Override
+    public void onChildDraw(@NonNull Canvas c, @NonNull RecyclerView recyclerView, @NonNull RecyclerView.ViewHolder viewHolder, float dX, float dY, int actionState, boolean isCurrentlyActive) {
+        super.onChildDraw(c, recyclerView, viewHolder, dX, dY, actionState, isCurrentlyActive);
+        if (actionState == ItemTouchHelper.ACTION_STATE_SWIPE && isCurrentlyActive) {
+            View itemView = viewHolder.itemView;
+
+            ColorDrawable background = new ColorDrawable();
```

Comment:
Move all local variables to top of scope (i.e. define every local variable below where also the View itemView is defined) and below that perform actions.